app/src/components/search/filters/selectFilter.tsx

```diff
@@ -203,8 +200,12 @@ const SelectFilterComponent = forwardRef<
         accordionData={[
           {
             accordionType: "default",
-            buttonInteractionRef: mergedRef,
-            label: capitalize(filter.name),
+            buttonInteractionRef: accordionButtonRef,
+            label: (
+              <Box noOfLines={1}>{`${capitalize(filter.name)}${
+                selected ? `: ${capitalize(selected.name)}` : ``
+              }`}</Box>
+            ),
```

Comment:
As discussed in person, you can refactor the `FilterAccordionComponent` so that it only every takes one object value since it won't ever render more than one for DCF.